web/templates/recent_view_row.hbs

```diff
@@ -4,12 +4,12 @@
             <div class="left_part recent_view_focusable" data-col-index="{{ column_indexes.stream }}">
                 {{#if is_private}}
                 <span class="zulip-icon zulip-icon-user"></span>
-                <a href="{{pm_url}}">{{t "Direct messages" }}</a>
+                <a href="{{pm_url}}" class="pm-url">{{t "Direct messages" }}</a>
                 {{else}}
                 <span class="stream-privacy-original-color-{{stream_id}} stream-privacy filter-icon" style="color: {{stream_color}}">
                     {{> stream_privacy }}
                 </span>
-                <a href="{{topic_url}}">{{stream_name}}</a>
+                <a href="{{topic_url}}" class="topic-url">{{stream_name}}</a>
```

Comment:
Not sure what should be the class used here. The text shown is stream name but the link points to topic narrow.

Reply:
I just updated it to use `stream-name`.